recipe/meta.yaml

```diff
@@ -44,6 +44,8 @@ test:
   # https://github.com/conda-forge/staged-recipes/pull/292#issuecomment-208080893.
   commands:                # [not win]
     - nosetests zmq.tests  # [not win]
+  imports:                 # [win]
+    - zmq                  # [win]
```

Comment:
That's right. We don't build `zeromq` for Windows at conda-forge. Yeah, I'm ok to go ahead with this. Maybe there is something that changed about the `defaults` copy of `zeromq`.